JSON_serializer/const.py

```diff
@@ -0,0 +1,2 @@
+ANSWERS = ['y', 'Y', 'YES', 'yes', 'Yes']
```

Comment:
this is not just answers, it's like positive answers, so `POSITIVE_ANSWERS` sounds better, isn't it?

Reply:
Agree, i'll correct it.